### DISCUSSION TITLE
Update readme compatibility for RN < 60

### DISCUSSION THREAD

**Discussion by 1Jesper1** [software-mansion/react-native-gesture-handler#1545] [General]
As seen in https://github.com/software-mansion/react-native-gesture-handler/releases/tag/1.4.0
`It is possible to use this release with RN < 0.60 but one have to rely on reverse-jetifying of gesture handler library. Read more on that here https://github.com/mikehardy/jetifier#to-reverse-jetify--convert-node_modules-dependencies-to-support-libraries`

Please include this in the Readme.
Till 1.10.3 it seems to work with RN 0.59.10.

**Comment by jgonet** [comment]
#1548

**Reply by 1Jesper1** [reply]
Nice!
